hybrasyl/Objects/ItemObject.cs

```diff
@@ -157,7 +157,7 @@ public ItemObjectType ItemObjectType
 
         public List<Xml.CastModifier> CastModifiers => Template.Properties.CastModifiers;
 
-        public uint MaximumDurability => Template.Properties.Physical.Durability;
+        public uint MaximumDurability => Template.Properties?.Physical?.Durability ?? 1;
```

Comment:
We should have maximum durability = 0 if not present, as 1 shows the 1/1 on objects without it defined.